lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -21,314 +21,290 @@
   name:
     required: true
     description:
-    - Service name
+      - Service name
   image:
     required: true
     description:
-    - Service image path and tag.
-      Maps docker service IMAGE parameter.
+      - Service image path and tag.
+        Maps docker service IMAGE parameter.
   state:
     required: true
     default: present
     description:
-    - Service state.
+      - Service state.
     choices:
-    - present
-    - absent
+      - present
+      - absent
   args:
     required: false
-    default: []
     description:
-    - List comprised of the command and the arguments to be run inside
-    - the container
+      - List comprised of the command and the arguments to be run inside
+      - the container
```

Comment:
This is not a new paragraph :)
```suggestion
        the container
```

Reply:
It's not :) I also fixed the actual text since it was wrong stating that it's compromised of both command and argument (extra confusing now that we actually have a `command` option).